src/recommender.py

```diff
@@ -459,13 +459,18 @@ def execute(self, arguments=None, persist=True, check_license=False):
                 'manifest_file_path': manifest_file_path
             }
             new_arr = [r['package'] for r in resolved]
+            # If new_err is empty list
+            if not new_arr:
+                recommendations.append(recommendation)
+                continue
```

Comment:
@sara-02 Have you added a unit test step to check this change?

Reply:
No, just locally. I add that as well.